ui/package.json

```diff
@@ -32,6 +32,7 @@
     "@types/node": "^20.12.12",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
+    "axios": "^1.7.2",
```

Comment:
No need for axios, just use fetch

Reply:
Using it because now creds need to be injected in every request to flow-api, so just created a new instance of axios with relevant headers. Check `ui/rpc/http.ts`

Reply:
np, make a light wrapper around fetch